server/server.go

```diff
@@ -0,0 +1,85 @@
+package server
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+	"regexp"
+
+	"github.com/diamondburned/arikawa/v3/api/webhook"
+	"github.com/diamondburned/arikawa/v3/discord"
+	"github.com/diamondburned/arikawa/v3/utils/sendpart"
```

Comment:
Use `libdb.so` Over `github.com/diamondburned`.

Reply:
This won't work. See https://github.com/diamondburned/arikawa/blob/v3/go.mod#L1.

Reply:
> This won't work. See https://github.com/diamondburned/arikawa/blob/v3/go.mod#L1.

Thanks for the heads up!